### DISCUSSION TITLE
What RNG Generator is used?

### DISCUSSION THREAD

**Discussion by Real-Yeeted-Bean** [jagrosh/GiveawayBot#158] [Questions] [answered]
Is it Uniform or Gaussian?

**Comment by MichailiK** [comment, marked as answer]
GiveawayBot uses [`java.security.SecureRandom`](https://docs.oracle.com/javase/8/docs/api/java/security/SecureRandom.html). The algorithm used depends on the software and hardware installed. SecureRandom will use whatever software or hardware component it deems best for random number generation.
